src/raytracer/verifyPath.m

```diff
@@ -5,58 +5,64 @@
+% should note the date and nature of any such change. Please explicitly  
+% acknowledge the National Institute of Standards and Technology as the 
+% source of the software.
+% 
 % NIST-developed software is expressly provided "AS IS." NIST MAKES NO
-% WARRANTY OF ANY KIND, EXPRESS, IMPLIED, IN FACT OR ARISING BY
-% OPERATION OF LAW, INCLUDING, WITHOUT LIMITATION, THE IMPLIED
-% WARRANTY OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE,
-% NON-INFRINGEMENT AND DATA ACCURACY. NIST NEITHER REPRESENTS
-% NOR WARRANTS THAT THE OPERATION OF THE SOFTWARE WILL BE
-% UNINTERRUPTED OR ERROR-FREE, OR THAT ANY DEFECTS WILL BE
-% CORRECTED. NIST DOES NOT WARRANT OR MAKE ANY REPRESENTATIONS
-% REGARDING THE USE OF THE SOFTWARE OR THE RESULTS THEREOF,
-% INCLUDING BUT NOT LIMITED TO THE CORRECTNESS, ACCURACY,
-% RELIABILITY, OR USEFULNESS OF THE SOFTWARE.
+% WARRANTY OF ANY KIND, EXPRESS, IMPLIED, IN FACT OR ARISING BY OPERATION  
+% OF LAW, INCLUDING, WITHOUT LIMITATION, THE IMPLIED WARRANTY OF 
+% MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE, NON-INFRINGEMENT AND 
+% DATA ACCURACY. NIST NEITHER REPRESENTS NOR WARRANTS THAT THE OPERATION OF 
+% THE SOFTWARE WILL BE UNINTERRUPTED OR ERROR-FREE, OR THAT ANY DEFECTS 
+% WILL BE CORRECTED. NIST DOES NOT WARRANT OR MAKE ANY REPRESENTATIONS  
+% REGARDING THE USE OF THE SOFTWARE OR THE RESULTS THEREOF, INCLUDING BUT 
+% NOT LIMITED TO THE CORRECTNESS, ACCURACY, RELIABILITY, OR USEFULNESS OF 
+% THE SOFTWARE.
 %
 % You are solely responsible for determining the appropriateness of using
-% anddistributing the software and you assume all risks associated with its use, including
-% but not limited to the risks and costs of program errors, compliance with applicable
-% laws, damage to or loss of data, programs or equipment, and the unavailability or
-% interruption of operation. This software is not intended to be used in any situation
-% where a failure could cause risk of injury or damage to property. The software
-% developed by NIST employees is not subject to copyright protection within the United
-% States.
+% and distributing the software and you assume all risks associated with  
+% its use, including but not limited to the risks and costs of program 
+% errors, compliance with applicable laws, damage to or loss of data, 
+% programs or equipment, and the unavailability or interruption of 
+% operation. This software is not intended to be used in any situation  
+% where a failure could cause risk of injury or damage to property. The 
+% software developed by NIST employees is not subject to copyright 
+% protection within the United States.
 %
-% Modified by: Mattia Lecci <leccimat@dei.unipd.it>, Used MATLAB functions instead of custom ones,
-%    supporting mobility check, improved if/else chain
+% Modified by: Mattia Lecci <leccimat@dei.unipd.it>, Used MATLAB functions 
+% instead of custom ones, supporting mobility check, improved if/else chain
 
 
 switch3 = 1;
```

Comment:
rename

Reply:
done

Reply:
i'd suggest pathExists instead of isPathExist

Reply:
done